hbase-common/src/main/java/org/apache/hadoop/hbase/HConstants.java

```diff
@@ -661,10 +660,8 @@
    * Timestamp to use when we want to refer to the oldest cell.
    * Special! Used in fake Cells only. Should never be the timestamp on an actual Cell returned to
    * a client.
-   * @deprecated Should not be public since hbase-1.3.0. For internal use only. Move internal to
-   *   Scanners flagged as special timestamp value never to be returned as timestamp on a Cell.
    */
-  @Deprecated
+  @InterfaceAudience.Private
   public static final long OLDEST_TIMESTAMP = Long.MIN_VALUE;
```

Comment:
Let's move it to another place?

Reply:
Makes sense. I looked through the usages and there doesn't seem to be a common place. Do we have a place for kind of "private" constants?

Reply:
Didn't found a good place for this one and created a new class for constants intended for private use.